ui/.env.example

```
@@ -44,6 +44,4 @@ VITE_UI_BG_MEDIUM=#374151                     # Gray-700
 VITE_UI_BORDER_COLOR=rgba(59,130,246,0.2)     # Blue border with opacity
 VITE_UI_TEXT_SECONDARY=#9ca3af                # Gray-400 for secondary text
 
-# Cosmos Client
-VITE_COSMOS_RPC_URL=http://localhost:26657
-VITE_COSMOS_REST_URL=http://localhost:1317
\ No newline at end of file
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)